example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_ActivityLogTestJetpack.kt

```diff
@@ -128,9 +150,9 @@ class ReleaseStack_ActivityLogTestJetpack : ReleaseStack_Base() {
 
         this.mCountDownLatch = CountDownLatch(1)
 
-        val firstActivity = activityLogModel(1)
-        val secondActivity = activityLogModel(2)
-        val thirdActivity = activityLogModel(3)
+        val firstActivity = activityLogModel(1, true)
```

Comment:
🔍 Nitpick: We usually add explicit parameter names to boolean parameters.
```suggestion
        val firstActivity = activityLogModel(1, rewindable  = true)
```

Reply:
This makes sense, thanks for pointing that out @malinajirka ! 🌟 

I am also on the fence here, not sure which is better. Having that as such for tests and using the IDE to see the parameter name sometimes is enough, but this is very subjective. For production code I would also be more strict with that.

![image](https://user-images.githubusercontent.com/9729923/104435027-977cee80-5594-11eb-9e8e-017da571e0a7.png)

In that case, and since the `index` parameter is something that would benefit from the explicit parameter names I would convert them both or non, just to have that consistency in place. How do you feel about that?

Reply:
> Having that as such for tests and using the IDE to see the parameter name sometimes is enough

Some devs have these IDE parameters disabled. I'm not sure if they can be enabled if you are using eg. vim. Moreover, they are not visible on GitHub and a lot of us do most of the reviews here. So imo we shouldn't rely on this behavior of the default AS's editor. Wdyt?

> In that case, and since the index parameter is something that would benefit from the explicit parameter names I would convert them both or non, just to have that consistency in place. How do you feel about that?

I agree, I'd change both.

Reply:
> Some devs have these IDE parameters disabled. I'm not sure if they can be enabled if you are using eg. vim. Moreover, they are not visible on GitHub and a lot of us do most of the reviews here. So imo we shouldn't rely on this behavior of the default AS's editor. Wdyt?

True that, I was basing a lot of what I said on my development setup. 👍 

> I agree, I'd change both.

Great, will do this change asap and let you know! 🙏

Reply:
👋 @malinajirka !

This is now done: a474430b889c9b225be57d6f3ff61ae4c8ffd3d6